Sources/Tools/plNetLog/ChunkBuffer.h

```diff
@@ -149,7 +149,7 @@ class ChunkBuffer
     QMutex              m_mutex;
     std::list<Buffer>   m_chunks;
 
-    void waitOnData(QMutexLocker& lock);
+    void waitOnData(QMutexLocker<QMutex>& lock);
```

Comment:
It looks like `QMutexLocker` was changed to a template only in Qt6, so this change will break Qt5.

Reply:
Ah, good catch, I guess we'll need some conditional macro alias for it :\